src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -82,24 +82,6 @@ public boolean isValidURI(String URI)
 		}
 	}
 
-
-	//Function to merge two arrays into third array.
-	public static <T> T[] concat(T[] a, T[] b) {
-		final int alen = a.length;
-		final int blen = b.length;
-		if (alen == 0) {
-			return b;
-		}
-		if (blen == 0) {
-			return a;
-		}
-		@SuppressWarnings("unchecked") final T[] result = (T[]) java.lang.reflect.Array.newInstance(a.getClass().getComponentType(), alen + blen);
-		System.arraycopy(a, 0, result, 0, alen);
-		System.arraycopy(b, 0, result, alen, blen);
-		return result;
-	}
```

Comment:
Careful: this is public, so it could be used by plugins ⇒ unused info doesn’t tell us much. That’s annoying, but fixing it will take a long time.

(we need a clearly specified plugin API to avoid this, but that currently does not exist, even though there’s the HighLevelSimpleClient, because plugins can access the full node)

⇒ please restore the function, otherwise plugins we don’t know may break.

Reply:
OK, re-added and marked Deprecated.